CMakeLists.txt

```diff
@@ -195,12 +192,17 @@ install(TARGETS dronecore
 
 install(FILES
     include/dronecore.h
-    include/device.h
-    ${CMAKE_CURRENT_BINARY_DIR}/include/device_plugin_container.h
+    core/device.h
     ${plugin_header_paths}
     DESTINATION "include/dronecore"
 )
 
+set(dronecore_install_include_dir "include/dronecore")
+set(dronecore_install_lib_dir ${lib_path})
+
+add_subdirectory(plugins/action)
+add_subdirectory(plugins/telemetry)
```

Comment:
Are we adding these plugins to show as example or is it required ?

Reply:
I believe they are in the root CMakeLists.txt just so their libraries get built, and also because they are required by the integration tests. But somebody writing his own executable can choose which library to link (at least `libdronecore.so`, obviously).

Reply:
That would make sense. I'm just making sure :-)

Reply:
OK. As I understand, we invoke CMake build for all the existing plugins in the directory [plugins/](https://github.com/dronecore/DroneCore/tree/develop/plugins). Those that are developed later will be built separately in their respective plugin directory.
Application which wants to use plugins need to link corresponding `dronecore_<plugin>.so` + `libdronecore.a`.

Reply:
@shakthi-prashanth-m correct, but it would probably be all `.so`.